compiler/dyno/include/chpl/uast/prim-ops-list.h

```diff
@@ -150,6 +150,7 @@ PRIMITIVE_G(GPU_GRIDDIM_Y, "gpu gridDim y")
 PRIMITIVE_G(GPU_GRIDDIM_Z, "gpu gridDim z")
 PRIMITIVE_G(GPU_ALLOC_SHARED, "gpu allocShared")
 PRIMITIVE_G(GPU_SYNC_THREADS, "gpu syncThreads")
+PRIMITIVE_G(GPU_SET_BLOCKSIZE, "gpu set blockSize")
```

Comment:
This should be `PRIMITIVE_R`. See https://github.com/chapel-lang/chapel/blob/main/compiler/dyno/include/chpl/uast/prim-ops-list.h#L29-L32

Reply:
Fixed in 9d559a8ac3b726ecaf63d09af50ae5ddac2eeafc